libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java

```diff
@@ -47,6 +52,10 @@
 
     private static boolean mIsToolbarExpanded = false;
 
+    private static final String[] sPermissions = new String[]{Manifest.permission.CAMERA,
```

Comment:
Let's rename this variable here to something more descriptive like `sCameraAndExternalStoragePermissions`

Reply:
Fixed.